core/src/main/scala/io/gearpump/util/Constants.scala

```diff
@@ -123,4 +123,8 @@ object Constants {
     classOf[HashPartitioner],
     classOf[ShuffleGroupingPartitioner],
     classOf[ShufflePartitioner])
+
+  //Security related
```

Comment:
Could we have more information here like what kind of security and how the duo are supposed to be used ?